apps/src/code-studio/dancePartySongArtistTags.js

```diff
@@ -1,7 +1,9 @@
-// Dance Party song titles mapped to the artist's Twitter handle to use in the // ShareDialog share links. Not all songs are available in all environments,
-// and the full list of in-use Dance Party songs can be found at: https://s3.console.aws.amazon.com/s3/buckets/cdo-sound-library/hoc_song_meta/?region=us-east-1&tab=overview
-//Full list of songs including potential (reflected here) can be found at:
-// https://tinyurl.com/y9r5xx5d
```

Comment:
I removed this part of the comment because this doc hadn't been used since the first year of Dance Party (2018)